tests/ui/resolve/issue-7663.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(unused_imports, dead_code)]
+
+mod test1 {
```

Comment:
Suggestion: can you rename these test modules as something more semantically meaningful, e.g.

- `test1` -> `glob_vs_glob_ambiguity`
- `test2` -> `non_glob_vs_non_glob_name_collision`